Source/Core/DiscIO/DirectoryBlob.cpp

```diff
@@ -19,69 +21,64 @@
 #include "Common/FileUtil.h"
 #include "Common/Logging/Log.h"
 #include "Common/StringUtil.h"
+#include "Common/Swap.h"
+#include "Core/Boot/DolReader.h"
 #include "DiscIO/Blob.h"
-#include "DiscIO/Enums.h"
-#include "DiscIO/Volume.h"
-#include "DiscIO/VolumeDirectory.h"
+#include "DiscIO/DirectoryBlob.h"
 
 namespace DiscIO
 {
 static u32 ComputeNameSize(const File::FSTEntry& parent_entry);
 static std::string ASCIIToUppercase(std::string str);
 static void ConvertUTF8NamesToSHIFTJIS(File::FSTEntry& parent_entry);
 
-const size_t VolumeDirectory::MAX_NAME_LENGTH;
-const size_t VolumeDirectory::MAX_ID_LENGTH;
+constexpr u64 GAME_PARTITION_ADDRESS = 0x50000;
+constexpr u64 PARTITION_TABLE_ADDRESS = 0x40000;
+static const std::array<u32, 10> PARTITION_TABLE = {
```

Comment:
btw, `static` doesn't need to be specified on file-scope variables that are const, since const has internal linkage by default.